src/ItemGrid.cpp

```diff
@@ -316,11 +318,16 @@ int cItemGrid::AddItem(cItem & a_ItemStack, bool a_AllowNewStacks, int a_Priorit
 	if ((a_PrioritarySlot != -1) && !IsValidSlotNum(a_PrioritarySlot))
 	{
 		LOGWARNING("%s: Invalid slot number %d out of %d slots",
-			__FUNCTION__, a_PrioritarySlot, m_NumSlots
```

Comment:
Can you fix "prioritary" while you're here?

Reply:
Done